.gitignore

```diff
@@ -26,6 +26,7 @@ test.tf
 *.test
 *.iml
 .vscode
+.env
```

Comment:
Why is this change required? ignore of development environment can be done in a gitignore_global